src/App.jsx

```diff
@@ -20,6 +21,8 @@ function App() {
           path="/article/:article_id/comments"
           element={<ArticleComments />}
         />
+        <Route path="/topics" element={<Topics />} />
+        <Route path="/topics/:topic_name/articles" element={<ArticlesByTopic />} />
```

Comment:
Could this be handled on `/articles` and use searchParams to differentiate between what you want?